### PR TITLE
Improve existing docstrings and add missing ones.

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Used these tutorials:
https://stackoverflow.com/questions/28763257/jsdoc-return-object-structure
http://usejsdoc.org/tags-type.html

### CODE REVIEW

**Comment by mykola-mokhnach** [comment]
There are still several undocumented methods there, but I really cannot fully understand how they work %)

File: lib/tools/adb-commands.js [hunk 1]
```diff
@@ -47,8 +47,8 @@ methods.initZipAlign = async function () {
 /**
  * Retrieve the API level of the device under test.
  *
- * @return (string) The API level as integer number, for example 21 for
- * Android Lollipop. The result of this method is cached, so all the further
+ * @return {string} The API level as integer number, for example 21 for
```
Comment (jlipps) [inline lib/tools/adb-commands.js]: maybe `integer string` rather than `integer number`? (since it's a string)

Comment (jlipps) [inline lib/tools/adb-commands.js]: ...should we return a number instead?

Comment (mykola-mokhnach) [inline lib/tools/adb-commands.js]: Maybe... Maybe it even makes sense to cache this value.
But due to some historical reasons unknown to me this method is implemented this way.

**Pull request merged**